sql/core/src/test/resources/sql-tests/inputs/postgreSQL/join.sql

```diff
@@ -743,20 +743,20 @@ select * from a left join b on i = x and i = y and x = i;
 --
 -- test NULL behavior of whole-row Vars, per bug #5025
 --
-select t1.q2, count(t2.*)
+select t1.q2, count(t2.q1, t2.q2)
```

Comment:
`count(t2.*)` means `count(1)` in pgsql?

Reply:
Yes, in pgsql
```
select * from (select 1 as a, null as b) as t1 join (select 1 as a, 'c' as b) as t2 on t1.a = t2.a
output:

a  | b      | a  | b
-- | --     | -- | --
1  | (null) | 1  | c

select count(t1.*) from (select 1 as a, null as b) as t1 join (select 1 as a, 'c' as b) as t2 on t1.a = t2.a
output:

count
--
1
``

Reply:
in spark, the second query will output 0

Reply:
since it's pgsql test, shall we use `count(1)` to match pgsql test intention?

Reply:
+1 for the @cloud-fan advice. Also, could you leave some comment there about why we change the original test query?

Reply:
@cloud-fan , @maropu , I did more tests on pgsql, seems it is much more wired than I thought.
TLDR: in pgsql, `count(t2.*)` means how many MATCHING rows from t2 table. and spark doesn't have such a semantic. Here, if we want to keep the output unchanged. it should be `count(t2.q1, t2.q2)`

Here is the example:
```
CREATE TABLE INT8_TBL AS SELECT * FROM (VALUES (1, 2), (2, 1), (3, NULL), (NULL, NULL)) AS v(q1, q2);
select * from int8_tbl as t1 left join int8_tbl as t2 on t1.q1 = t2.q2;

t1.q1  | t1.q2  | t2.q1  | t2.q2
--     | --     | --     | --
1      | 2      | 2      | 1
2      | 1      | 1      | 2
3      | (null) | (null) | (null)
(null) | (null) | (null) | (null)

select count(t1.*), count(t2.*) from int8_tbl as t1 left join int8_tbl as t2 on t1.q1 = t2.q2;

t1.count | t2.count
--       | --
4        | 2

```